spec/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser_spec.rb

```diff
@@ -7,6 +7,8 @@
     VCR.insert_cassette("#{vcr_path}/mock_cabinet", options)
     VCR.insert_cassette("#{vcr_path}/mock_config_patterns", options)
     VCR.insert_cassette("#{vcr_path}/mock_switches", options)
+    VCR.insert_cassette("#{vcr_path}/mock_compliance_policy", options)
+
```

Comment:
@CharlleDaniel can you remove this line?

Reply:
yes, I can. Thanks